Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -192,13 +193,14 @@ namespace ROS2
                 report += "`";
             }
             m_checkUrdfPage->ReportURDFResult(report, urdfParsedSuccess, urdfParsedWithWarnings);
-            if (parsedUrdfOutcome.m_parseMessages.size() > 0)
+            if (!parsedSdfOutcome.GetParseMessages().empty())
             {
                 report += "\n\n";
-                report += tr("URDF parser returned following messages:") + "\n\n";
+                report += tr("URDF/SDF parser returned following messages:") + "\n\n";
                 report += "```bash\n";
-                report += QString::fromUtf8(parsedUrdfOutcome.m_parseMessages.data(), int(parsedUrdfOutcome.m_parseMessages.size()));
+                report += QString::fromUtf8(parsedSdfOutcome.GetParseMessages().data(), int(parsedSdfOutcome.GetParseMessages().size()));
                 report += "\n```\n";
+                AZ_Printf("RobotImporterWidget", "SDF Stream: %s", parsedSdfOutcome.GetParseMessages().c_str());
```

Comment:
```suggestion
                auto parsedMessages = parsedSdfOutcome.GetParseMessages();
                AZ_Printf("RobotImporterWidget", "SDF Stream: %.*s", AZ_STRING_ARG(parsedMessages));
```

Reply:
This is not really gaining us anything.
That being said, a newline is needed here.